app/src/main/java/com/example/numberbaseball/base/BaseActivity.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.numberbaseball.base
+
+import android.os.Bundle
+import androidx.annotation.LayoutRes
+import androidx.appcompat.app.AppCompatActivity
+
+/**
+ * Created by jiwun on 2019-08-23.
+ */
+abstract class BaseActivity(@LayoutRes val layoutId:Int) : AppCompatActivity() {
```

Comment:
클래스 관련 코드 컨벤션을 지켜주세요.
```suggestion
abstract class BaseActivity(
@LayoutRes 
val layoutId:Int
) : AppCompatActivity() {
```
https://kotlinlang.org/docs/reference/classes.html#derived-class-initialization-order

Reply:
[ac67522] 완료했습니다.